scalyr_agent/configuration.py

```diff
@@ -1057,6 +1057,13 @@ def include_raw_timestamp_field(self):
         message as recorded by those systems."""
         return self.__get_config().get_bool("include_raw_timestamp_field")
 
+    @property
+    def merge_json_parsed_lines(self):
+        """If True, and events are getting parsed with the parse_as_json feature, events with messages that do not
+        end in a newline character will be joined together until a newline character is found. This is to work around
```

Comment:
You should also probably mention that the timestamp for the first line (among all of those joined) will be used for the joined line's timstamp.